### DISCUSSION TITLE
Stop sync to delete files locally without deleting on OneDrive?

### DISCUSSION THREAD

**Discussion by ooverbee** [abraunegg/onedrive#2299] [Q&A] [answered]
I've run out of space on Ubuntu (22.04) and I would like to delete some files that I've synced locally from OneDrive. But I don't want to delete them from my OneDrive, I just want to delete them from my local Ubuntu storage. How can I do this? Currently, every change I make on Ubuntu is visible across all my devices linked to OneDrive.

**Comment by abraunegg** [comment, marked as answer]
@ooverbee 
> But I don't want to delete them from my OneDrive, I just want to delete them from my local Ubuntu storage. How can I do this?

Your best option here is to 'move' the files that you do not want to sync locally to a 'new folder' within OneDrive online .. call this 'archive' or 'not-for-local-sync'.

For the application configuration, you would then use the 'skip_dir' option to skip syncing this folder you have created online.

The steps to achieve this would be:
* Stop your local sync processes | systemd services
* Make the change online
* Configure your client and test manually with `--dry-run`
* Sync manually to get everything back in-sync
* Re-enable any sync processes | systemd services


The alternative here however is also just buy a larger disk for your Linux system if possible .....
